CONTRIBUTING.md

```diff
@@ -46,3 +46,17 @@ On Windows run:
 ```
 gradlew.bat clean check
 ```
+
+# 🐞 Debug
+
+To debug the gradle-dep-tree process, follow these steps:
+1) Create a new debug configuration in your IDE. Choose "Remote JVM Debug" and ensure the following fields are correctly configured:
+   * Debugger Mode: Attach to remote JVM
+   * Host: localhost
+   * Post: 5005
+   * Use module classpath: <no module>
+2) Search for the point in your code that triggers the gradle-dep-tree process.
+3) Add the following flags to the command: "-Dorg.gradle.debug=true", "--no-daemon"
```

Comment:
I suggest combining 2 and 3 into one step:
2) Add the following flags to the Gradle command: `-Dorg.gradle.debug=true --no-daemon`
or
2) Run the Gradle command with the following flags: `-Dorg.gradle.debug=true --no-daemon`